package.json

```diff
@@ -21,8 +21,10 @@
     "istanbul-lib-coverage": "^1.0.0",
     "jasmine-reporters": "^2.2.0",
     "lerna": "2.0.0-beta.26",
+    "lint-staged": "^3.0.2",
```

Comment:
I don't think we actually want to use lint-staged within Jest at this point. At the least I'd prefer to decouple it from adding the related tests feature from adding lint-staged. Would you mind removing it again?